### ISSUE TITLE
[1.2.10][Ubuntu] Can not connect to server if sslCiphers is manually set

### ISSUE THREAD

**Issue by Hannobal** [mumble-voip/mumble#1763]
If I set the sslCiphers manually to the following, I can't connect to my server. 
``` shell
sslCiphers=ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384
```
Murmur server log says this:
> <W>2015-07-23 16:22:14.868 1 => <9:(-1)> Connection closed: Error during SSL handshake: error:1408A0C1:SSL routines:SSL3_GET_CLIENT_HELLO:no shared cipher [13]

On Windows 8.1 and OSX it works fine. Both are updated to latest version.

I am using Ubuntu 15.04 with latest updates and all of us are using Mumble 1.2.10.

**Comment by mkrautz** [comment]
The Linux clients use the system Qt 4 libraries. Qt 4 doesn't support TLS 1.2, I'm afraid.

Our own binaries use a patched Qt 4 which allows use of TLS 1.2 and ECDHE/DHE cipher suites.

The distro and PPA packages don't, and they aren't going to patch.

We're looking into providing binary packages for Fedora/Debian/Ubuntu ourselves, but we aren't there yet.

One workaround you could use is to use the snapshot PPA packages. That way, Mumble is built against Qt 5 and will be able to use modern TLS features.

**Comment by Hannobal** [comment]
I will wait for 1.3.0 :)

Any idea when will 1.3.0 release? Looks like that there is lots of work remaining.
